client/src/components/signup/Signup.js

```diff
@@ -7,23 +7,16 @@ import '../../css/signup-login.css'
 
 // component imports
 import SignupForm from './SignupForm'
-import SignupForm2 from './SignupForm2'
-
-import DribbbleSocial from './DribbbleSocial'
-import ListMyDribbbles from './ListMyDribbbles';
```

Comment:
@vdumouchel 

Is this important for anything? Alam said Akshay deleted it while working with him.